test/storybook.test.js

```diff
@@ -191,7 +192,7 @@ describe('percy storybook', () => {
       req.body.data.relationships.resources
         .data.find(r => r.attributes['is-root']).id
     ]).sort((a, b) => a[0] > b[0] ? 1 : -1)).toEqual([
-      ['foo: bar', sha(storyDOM)],
+      ['foo: bar', sha(storyDOM.replace('<head></head>', `<head>${polyfill}</head>`))],
```

Comment:
`percy.snapshot` received the html with injection. We can't have an assertion for the original HTML content since there's no reference to it. CLI goes to the page and gives us the HTML. Let us keep it this way

Reply:
Added